### ISSUE TITLE
CSS-Anpassung an Plugin The Events Calendar 

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/FAU-Einrichtungen#1228]
Wir planen das Plugin TheEventsCalandar auf unser Netzwerk anzubieten.
Damt es bei der AUsgabe in der Sidebar dieselbe Optik zeigt, wie das bisherige rrze-calendar-Plugin soll eine neue Plugin-SCSS-File angelegt werden, in der die Optik entsprechend überschrieben wird.
(Aber nur bei Nutzung in der Sidebar. SOnst ist die Optik von TheEventsCalendar ja recht gut).

**Comment by xwolfde** [comment]
Zum testen und entwickeln kann die freie Version von The Events Calendar als Grundlage genommen werden.

**Comment by MManthey** [comment]
> CSS-Ergänzung TEC für Darstellung in Sidebar

Das einzige Widget ( 'Events List'), dass von dem freien Plugin bereitgestellt wird, konnte ich problemlos in der Sidebar einbinden.

![image](https://user-images.githubusercontent.com/43784820/231137842-1e51ff9c-fb3b-4cb6-8d47-93a3ab83a4c7.png)

Für ein Einbinden per Shortcode wird meines Erachtens die Pro-Version des Plugins benötigt.

Wie kann ich die Probleme sonst reproduzieren?

**Comment by lukasniebler** [comment]
@MManthey Ich denke es geht tatsächlich erstmal nur um die Listenausgabe. Die sieht / sah bei uns auf der Instanz tatsächlich aktuell so aus: https://www.wordpress.rrze.fau.de/test-5/

Wir sind aber gerade auch nochmal im Austausch mit dem TEC-Support, da es allgemein noch einige Probleme zum TEC-Pro Plugin gibt

Wenn die Ansicht bei dir funktioniert, könnte es auch gut sein, dass die Pro-Version bei uns das Problem verursacht. Für den Anfang genügt die Listenansicht. (Damit ist es auch später leichter unsere Corporate Identity einzuhalten.)

**Comment by lukasniebler** [comment]
Jetzt auch nochmal mit Rücksprache mit Rolf, unserem Senior Developer.
Es reicht, wenn die Gratis-Version des Plugins gut aussieht. Noch ist nicht klar, ob wir wirklich die Pro-Version später zur Verfügung stellen. Dazu zählt dann die Listenausgabe und natürlich die Kalender-Ansichten (ohne Shortcode) auf der Event-Übersicht.

**Comment by FitoreGashi** [comment]
http://sites.chillibrain.de/events/ das ist die Listenausgabe - Gratis-Version des Plugins @lukasniebler 

Wir können versuchen, einen ShortCode zu erstellen, um die Liste in den Seitenleisten der Seiten zu verwenden

**Comment by lukasniebler** [comment]
@FitoreGashi – Ich habe jetzt mal Rücksprache gehalten. Wir werden auf unserem CMS in Zukunft auch nur die Standard-Version nutzen. Der Shortcode wäre sehr praktisch / gut, falls Ihr es schafft einen zu erstellen, der die Ausgabe der Liste in der Sidebar ermöglicht

**Comment by FitoreGashi** [comment]
![Screenshot 2023-04-19 at 09 09 27](https://user-images.githubusercontent.com/101202370/232994971-b3493a17-21aa-4267-9c2a-a00e57b371ac.png)
![Screenshot 2023-04-19 at 09 09 21](https://user-images.githubusercontent.com/101202370/232994977-0666bb22-84a9-4107-af2e-e189b9c2aaad.png)
Der Shortcode zeigt die Liste so an, ist das ok? @lukasniebler

**Comment by lukasniebler** [comment]
@FitoreGashi Von der Struktur her ist es super so. Aber kannst du beim Design noch etwas nachjustieren?
Bei den Boxen wäre es sehr schön, wenn sie dem jetzigen Design nahe kommen:
- Die Hintergrundfarbe in FAU-Blau
- Der Abstand über und unter dem Datum in der Box noch etwas erhöhen
- Den Abstand zwischen den Einträgen noch etwas erhöhen
- Den Border Radius auf den gleichen Wert wie bei den Buttons setzen, damit es einheitlicher ist?

So sieht das aktuelle Design aus:
<img width="370" alt="Bildschirm­foto 2023-04-19 um 09 15 06" src="https://user-images.githubusercontent.com/58593037/232996750-88bd3641-1aba-43b3-b31b-ad1940da104b.png">

Auf https://www.wordpress.rrze.fau.de kannst du die aktuelle Version sehen. (Ihr könnt dem ganzen aber gerne einen etwas moderneren Touch geben, falls Ihr Ideen habt).

**Comment by lukasniebler** [comment]
Also etwas mehr in diese Richtung:
<img width="1124" alt="Bildschirm­foto 2023-04-19 um 09 43 27" src="https://user-images.githubusercontent.com/58593037/233004526-5a77d684-4d0e-47a5-b4a3-8ca5169b121d.png">

**Comment by FitoreGashi** [comment]
Ja, wir können das Design anpassen, Ich schicke Ihnen später ein mockup

**Comment by FitoreGashi** [comment]
Dies ist unsere Version für den Listenkalender in der sidebaroder im Seiteninhalt. Ich werde Ihnen später auch die Ansicht der Veranstaltung und den vollständigen Kalender zusenden @lukasniebler 
![Screenshot 2023-04-19 at 10 56 33](https://user-images.githubusercontent.com/101202370/233024769-d715e8dd-6fda-46cf-b232-581a3837415e.png)

**Comment by lukasniebler** [comment]
@FitoreGashi Sieht sehr gut aus. Passt so.

Bei der Uhrzeit fällt später ja wahrscheinlich noch das "am" und "pm" weg, wenn die Kunden 24h-Format nutzen, richtig?

**Comment by FitoreGashi** [comment]
Ja, ich werde das Datumsformat auf H:i ändern 
![Screenshot 2023-04-19 at 11 45 56](https://user-images.githubusercontent.com/101202370/233036953-420337fa-8a95-4cea-97a9-73d27ca442e1.png)

**Comment by FitoreGashi** [comment]
![Screenshot 2023-04-19 at 12 43 11](https://user-images.githubusercontent.com/101202370/233052001-5ff1b434-f749-4f94-97ae-b314983c5e08.png)
![screencapture-localhost-event-new-2023-04-19-12_46_55](https://user-images.githubusercontent.com/101202370/233052026-c84b9871-6ca2-43fc-9b17-4de24fc6164e.png)
![screencapture-localhost-events-monat-2023-05-2023-04-19-12_42_52](https://user
![screencapture-localhost-events-liste-2023-04-19-12_42_26](https://user-images.githubusercontent.com/101202370/233052063-db5fdd9a-b4bd-489e-8951-ddbd85710847.png)
-images.githubusercontent.com/101202370/233052034-a2fb7735-41b5-44cc-8246-658648130b12.png)

In diesem Modell habe ich auch die Standardansichten des Kalenders geändert, um den Designstil und die Farben anzupassen

**Comment by lukasniebler** [comment]
Sieht gut aus 👍
